linked-lists/lib/linkedList.js

```diff
@@ -141,20 +171,18 @@ class LinkedList {
   }
 
   static mergeList(list1, list2) {

```

Comment:
The method mergeList() would be perfect to comment on its behavior.
The other methods (reverse, append, prepend, etc.) are pretty self-explanatory in their behavior as they are mostly synonymous across languages and CS concepts.  However, this mergeList() method behavior is not as obvious. Does it prepend a list, append a list, alternate a list, etc.? How does it handle a longer list with a shorter list? These behaviors are not clear unless one reads through the code. Consider leaving concise method descriptions on methods whose behaviors are not obvious from the name of the method alone.